packages/plugin-list/test/Checklist.test.ts

```diff
@@ -297,12 +300,14 @@ describePlugin(List, testEditor => {
                                     <li class="unchecked">3.2</li>
                                 </ul>
                             </li>
-                            <li class="unchecked">4</li>
+                            <li class="checked">4</li>
```

Comment:
So... the test used to be wrong then ? The comment in `contentBefore` uses the premise that "4 is checked" yet it used to be unchecked ?